### ISSUE TITLE
[Bug]: Neovim crashes on multiple instances, except 1

### ISSUE THREAD

**Issue by KoolieAid** [jiriks74/presence.nvim#35]
### Are you on the latest version?

- [X] I have updated to the latest version.

### Have you tried it with default config?

- [X] I have tried the default config.

### Description

The first instance of neovim crashes when another one opens. presence must be loaded on both instances

### Neovim version

```markdown
NVIM:0.9.5
Build type:RelWithDebInfo
LuaJIT:2.1.1703942320
```

### OS information

Windows 10

### Steps to reproduce

1. Open terminal
2. Open neovim
3. Open another terminal
4. Open neovim
5. Observe that the first instance crashes

### Logs

<details>

```
none
```

</details>

**Comment by KoolieAid** [comment]
Original: https://github.com/andweeb/presence.nvim/issues/132
I imported since it seems the original repo is unmaintained and someone already made an issue

**Comment by jiriks74** [comment]
I don't have a Windows machine right now but I'll look into it when I have the time to install it on something. 

Did you get at least a core dump? Even though I haven't messed around with them I could try to look into it if it were available.

**Comment by KoolieAid** [comment]
No, I didn't get a core dump, I did get a glance to the last message printed to the screen before it crashed, which is
![image](https://github.com/jiriks74/presence.nvim/assets/26061711/922876dd-91f9-4aea-bfb1-16b7bb9552db)

**Comment by jiriks74** [comment]
This should get you a proper log file @KoolieAid 

```
nvim -V10nvim.log
```
